.github/workflows/validate-protobufs.yml

```diff
@@ -22,7 +18,7 @@ jobs:
 
       - name: Update hedera-services branch
         run: |
-          sed -i 's/branch = "main"/branch = "${{ env.branch }}"/' hedera-services/hapi/build.gradle.kts
+          sed -i.bak "s/branch = \".*\"/branch = \"${GITHUB_HEAD_REF}\"/" hedera-services/hapi/build.gradle.kts
```

Comment:
Something worth noting:
Sed is greedy by default, so this might match a lot more than we want, basically it will extend as far as it can without crossing to a new line.  That _shouldn't_ be an issue, unless someone puts a comment at end of line with a `"` in it, in which case this may break (that's why I restricted the match set in my suggestion).